src/tridiagonal.rs

```diff
@@ -0,0 +1,801 @@
+//! Vectors as a TriDiagonal matrix
+//! &
+//! Methods for tridiagonal matrices
+
+use std::ops::{Index, IndexMut};
+
+use cauchy::Scalar;
+use ndarray::*;
+use num_traits::One;
+
+use super::convert::*;
+use super::error::*;
+use super::lapack::*;
+use super::layout::*;
+
+/// Represents a tridiagonal matrix as 3 one-dimensional vectors.
+/// This struct also holds the layout of the raw matrix.
+#[derive(Clone, PartialEq)]
+pub struct TriDiagonal<A: Scalar> {
```

Comment:
```suggestion
pub struct Tridiagonal<A: Scalar> {
```

"Tridiagonal" seems not to be a compound word

- https://mathworld.wolfram.com/TridiagonalMatrix.html
- https://en.wikipedia.org/wiki/Tridiagonal_matrix

Reply:
I renamed all the part of `TriDiagonal` to `Tridiagonal`.